instant-markdown-d

```diff
@@ -1,10 +1,10 @@
-#!/bin/sh
-':' //; exec "`command -v nodejs || command -v node`" "$0"
+#!/usr/bin/env node
```

Comment:
Yeah, I know this causes a problem with the way that Ubuntu packages node, but nodejs will not generate the instant-markdown-d.cmd script necessary for running the server on Windows if the shebang is not `/usr/bin/env node`. Perhaps the solution would be to have Ubuntu users make a symlink from nodejs to node?